README.md

```diff
@@ -2,15 +2,15 @@
 
 ## Nitric Examples
```

Comment:
Could you also work in the content from the current example apps readme? That includes a variety of key terms that we could help people find this based on their use case and searching.